### ISSUE TITLE
Tooltip Image on Series- Details page shows random Cover instead of currently displayed Image

### ISSUE THREAD

**Issue by t0815** [japanesemediamanager/jmmclient#244]
If you hover over a cover image in Series -details Page, the tooltip displays a random cover image each time, instead the currently displayed image.

![image_bug_1](https://cloud.githubusercontent.com/assets/10379828/9831321/07324c88-5952-11e5-9c27-7cde5fce03a0.jpg)

![image_bug_2](https://cloud.githubusercontent.com/assets/10379828/9831324/0e588b80-5952-11e5-902b-c11a9c549021.jpg)

**Comment by AreFinstad** [comment]
yeah it does this, i guess its because the Series list does load a random picture each time you load the series into view. not too bothered with this here myself as i use the Fanart view (uses more of the empty screen space) as the cover is already loaded on the series list on the left anyway.

also note that you have some weird cuts in your image, prob by cutting in paint/gimp/PS.
next time use ShareX as it makes things much faster and simpler for you when sharing shots, also less empty white areas.
https://getsharex.com/


also note to the OP, that you can checkout all the images of the series under the Images tab at the top of the detail panel where you can open the picture in your favorite image loader and so on.

**Comment by ElementalCrisis** [comment]
This is something I also noticed so I took a look. It's because both the image and tooltip are calling the same string, FanartPathThenPosterPath which selects a random image from the image list for that series. I took a shot at this but my beginner skills are not up to the task it seems, lol. I imagine a new string needs to be created that displays the results of string PosterPathWithRandoms instead of running through it.
